.github/workflows/build-test.yml

```diff
@@ -7,74 +7,114 @@
 #  Author: Michael E. Tryby
 #          US EPA - ORD/CESER
 #
+#          Caleb A. Buahin
+#          Xylem Inc.
+#
 
 name: Build and Test
 
 on:
   push:
-    branches: [ develop ]
+    branches: [develop]
   pull_request:
-    branches: [ develop ]
+    branches: [develop]
 
 jobs:
-  win_build:
-    runs-on: windows-2016
+  build:
+    runs-on: ${{ matrix.os }}
     continue-on-error: ${{ matrix.experimental }}
-
-    defaults:
-      run:
-        shell: cmd
-
+    env:
+      OMP_NUM_THREADS: 1
+      PROJECT: swmm
+      BUILD_HOME: build
+      TEST_HOME: nrtests 
+      NRTESTS_URL: https://github.com/OpenWaterAnalytics/swmm-nrtestsuite
     strategy:
       fail-fast: false
       matrix:
-        generator: [Visual Studio 15 2017]
-        experimental: [false]
+        os: [windows-2016, ubuntu-latest, macos-latest]
         include:
-          - generator: Visual Studio 15 2017 Win64
+          - os: windows-2016
+            generator: Visual Studio 15 2017 Win64
             experimental: true
+            script_extension: cmd
+            ci_tools_path: ci-tools/windows
```

Comment:
Set working directory instead.

Reply:
@michaeltryby , using working-directory screws with our ci-tools scripts since they refer to relative paths.